### PR TITLE
Add init

### PR DESCRIPTION
**Pull request by karam084**
## Project requirements

- Create a new React app called 'hello-react-front-end'
- Initialize your project with Git.
- Create the App component with react-router.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- Display the random greeting in your Greeting component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,183 @@
...
+- Acknowledgements
+- License
 
-## Available Scripts
 
-In the project directory, you can run:
+<div align="center">
 
-### `npm start`
+  <img src="./app/assets/images/recipe_erd_2_members.png" alt="logo" width="140"  height="auto" />
```
Comment (sja-thedude) [inline README.md]: - [x] Kindly remove the microverse logo as it's not required.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,183 @@
-# Getting Started with Create React App
+<a name="Hello-React-Front-End"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+REQUIRED SECTIONS:
+- Table of Contents
```
Comment (sja-thedude) [inline README.md:5]: - [ ] Kindly make sure to add the link to the backend repo as required.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @karam084, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have the following because of which it is invalidated. Check comments (on both frontend and backend PR).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (karam084) [reply README.md:5]: Thank you, Updateed

Comment (sja-thedude) [reply README.md:5]: Hi @karam084, You haven't added a link to the backend in your frontend repo?

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @karam084, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**